pacstall

```diff
@@ -180,133 +184,126 @@ while [[ "$1" =~ ^- && ! "$1" == "--" ]]; do
       -V --version, Prints pacstall version
       -L --list, Lists installed packages
       -Up --upgrade, Upgrades packages
-      -Qd --query-depends, Get dependencies of package
       -Qi --query-info, Get package info
        "
       exit 0
     ;;
 
     -I|--install)
-      export LC_ALL=C
-
       function trap_ctrlc () {
 	      fancy_message warn "the installation of $2 was interrupted, removing files"
-	      rm -rf /tmp/pacstall/*
+	      rm -rf "${SRCDIR:?}"/* # :? makes bash error out in case SRCDIR is empty, saving us from yonking /* directory by mistake
```

Comment:
The proper term is `yoink`, not `yonk`

Reply:
```suggestion
	      rm -rf "${SRCDIR:?}"/* # :? makes bash error out in case SRCDIR is empty, saving us from yoinking /* directory by mistake
```
Right